modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -25,59 +37,93 @@ By default, adds to coercion exceptions more information about data that fails t
 * Type: `Boolean`
 * Default: `true`
 
-//com.mulesoft.dw.date_minus_back_compatibility//
-|`com.mulesoft.dw.date_minus_back_compatibility`|
-Restores the xref:dataweave-cookbook-add-and-subtract-time.adoc[Add and Subtract Time] behaviors that changed in 2.3.0 when this property is set to `true`.
+
+//com.mulesoft.dw.cpulimit.watchdog//
+|`com.mulesoft.dw.cpulimit.watchdog`|
+Monitors and limits script execution time when set to `true`.
 
 * Type: `Boolean`
-* Default: `false`
+* Default: `true`
 
-Available for language levels: 2.3, 2.4
 
-Example:
+//com.mulesoft.dw.date_minus_back_compatibility//
+|`com.mulesoft.dw.date_minus_back_compatibility`|
+
+Restores the xref:dataweave-cookbook-add-and-subtract-time.adoc[Add and Subtract Time] behaviors that changed in 2.3.0 when this property is set to `true`.
+
+For example, consider the following DataWeave script:
 [source,dataweave,linenums]
 ----
 %dw 2.0
 output application/dw
 ---
 \|2019-10-01\| - \|2018-09-23\|
 ----
-
 * In DataWeave 2.3.0, the expression returns `PT8952H`.
 * In DataWeave 2.2.0, the expression returns `P1Y8D`.
 
+* Type: `Boolean`
+
+* Default: `false`
+
+Available for language levels: 2.3, 2.4, 2.5
+
+
+//com.mulesoft.dw.decode_binaries_with_writer_encoding//
+|`com.mulesoft.dw.decode_binaries_with_writer_encoding`|
+When set to true, binary values in JSON are written using JsonWriter encoding. Otherwise, the property uses either the encoding defined in the value schema or the default one.
```

Comment:
```suggestion
When set to `true`, binary values in JSON are written using JsonWriter encoding. Otherwise, the property uses either the encoding defined in the value schema or the default one.
```